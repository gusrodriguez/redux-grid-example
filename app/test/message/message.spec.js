
/* eslint-disable */ //disable lint because 'describe' does not need to be defined in this scope.
const React = require('react');
const { shallow, mount, render } = require('enzyme');
const Message = require('../../shared/components/message');
const renderer = require('react-test-renderer');

describe('Message Component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <Message type="alert-success" text="The new user has been added." />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
})