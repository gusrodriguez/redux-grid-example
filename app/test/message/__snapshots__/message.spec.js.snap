// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message Component renders correctly 1`] = `
<div
  className="alert alert-success alert-dismissible"
>
  <span
    aria-label="close"
    className="close"
    data-dismiss="alert"
    onClick={undefined}
  >
    Ã—
  </span>
  The new user has been added.
</div>
`;
